src/modules/dashboard/components/ColonyHome/ColonySubscription/ColonySubscription.css

```diff
@@ -45,28 +47,6 @@
   background-color: color-mod(var(--colony-blue) alpha(10%));
```

Comment:
Don't forget `border: 0px` here as this one doesn't have its border on hover

Reply:
Thanks for catching this Will. I thought it would make more sense to have consistency for all usages of the button since in the other cases we do use the border. Maybe this can be a good questions for @arrenv. What do you think, should we keep the border or is there a specific reason why this one has no border?

<img width="294" alt="Screen Shot 2022-07-27 at 7 21 55 AM" src="https://user-images.githubusercontent.com/71563622/181160957-d9066a4d-1d67-4ebb-b5a4-f01d11978532.png">
?

Reply:
> should we keep the border or is there a specific reason why this button has no border?

Yes, we can keep the border.